src/Provision.js

```diff
@@ -0,0 +1,313 @@
+    client.AddContentLibraryGroup({libraryId, groupAddress: contentAdmins, permission: "contributor"}),
+
+    // Content users
+    client.AddContentLibraryGroup({libraryId, groupAddress: contentUsers, permission: "accessor"})
+  ];
+
+  await Promise.all(promises);
+};
+
+const SetObjectPermissions = async (client, objectId, tenantAdmins, contentAdmins, contentUsers) => {
+  let promises = [
+    // Tenant admins
+    client.AddContentObjectGroupPermission({objectId, groupAddress: tenantAdmins, permission: "see"}),
+    client.AddContentObjectGroupPermission({objectId, groupAddress: tenantAdmins, permission: "access"}),
+    client.AddContentObjectGroupPermission({objectId, groupAddress: tenantAdmins, permission: "manage"}),
+
+    // Content admins
+    client.AddContentObjectGroupPermission({objectId, groupAddress: contentAdmins, permission: "see"}),
+    client.AddContentObjectGroupPermission({objectId, groupAddress: contentAdmins, permission: "access"}),
+    client.AddContentObjectGroupPermission({objectId, groupAddress: contentAdmins, permission: "manage"}),
+
+    // Content users
+    client.AddContentObjectGroupPermission({objectId, groupAddress: contentUsers, permission: "see"}),
+    client.AddContentObjectGroupPermission({objectId, groupAddress: contentUsers, permission: "access"})
+  ];
+
+  await Promise.all(promises);
+};
+
+const InitializeTenant = async ({client, kmsId, tenantName, debug=false}) => {
+  let tenantAdminId = await client.userProfileClient.TenantId();
+  let tenantAdminSigner = client.signer;
+
+  if (!tenantAdminId){
+    throw Error("No tenant admin group set for account.");
+  }
+
+  let tenantAdminGroupAddress = client.utils.HashToAddress(tenantAdminId);
+
+  const contentAdminGroupAddress = await client.CreateAccessGroup({
+    name: `${tenantName} Content Admins`
+  });
+
+  const contentUserGroupAddress = await client.CreateAccessGroup({
+    name: `${tenantName} Content Users`
+  });
+
+  const tenantSlug = tenantName.toLowerCase().replace(/ /g, "-");
+
+  await client.AddAccessGroupManager({
+    contractAddress: contentAdminGroupAddress,
+    memberAddress: tenantAdminSigner.address
+  });
+
+  await client.AddAccessGroupManager({
+    contractAddress: contentUserGroupAddress,
+    memberAddress: tenantAdminSigner.address
+  });
+
+
+  if (debug){
+    console.log("\nTenant Admin Groups ID:\n");
+    console.log("\t", await client.userProfileClient.TenantId());
+
+    console.log("\nAccess Groups:\n");
+    console.log(`\tOrganization Admins Group: ${tenantAdminGroupAddress}`);
+    console.log(`\tContent Admins Group: ${contentAdminGroupAddress}`);
+    console.log(`\tContent Users Group: ${contentUserGroupAddress}`);
+  }
+
+  let groups = {
+    tenantAdminGroupAddress,
+    contentAdminGroupAddress,
+    contentUserGroupAddress
+  };
+
+  /* Content Types - Create Title, Title Collection and Production Master and add each to the groups */
+
+  const typeMetadata = {
```

Comment:
can you make it so Production Master doesn't have display app set? Michael filed an issue for this in elv-client-js

https://github.com/eluv-io/elv-client-js/issues/155

Reply:
Remove it from the typeMetadata like this?
```
  const typeMetadata = {
    bitcode_flags: "abrmaster",
    bitcode_format: "builtin",
    public: {
      "eluv.manageApp": "default",
    }
  };
 ```